snap/snapcraft.yaml

```diff
@@ -26,6 +26,10 @@ apps:
     slots:
       - wayland
       - x11
+    plugs:
```

Comment:
Are these plugs required to be listed here? Isn't the entries at the bottom of the file sufficient?

Reply:
That's a good question... should I presume that the entries at the bottom are "global" ones, and the entries in each part are specific for that part? Because in that case, the ones at the bottom are the ones that should be removed...

Reply:
Fixed.